service/bfg/bfg.go

```diff
@@ -565,7 +565,7 @@ func (s *Server) handleBitcoinUTXOs(ctx context.Context, bur *bfgapi.BitcoinUTXO
 	return buResp, nil
 }
 
-var ErrAlreadyProcessed error = fmt.Errorf("Already Processed BTC Block")
+var ErrAlreadyProcessed = errors.New("already processed BTC block")
```

Comment:
btc should be spelled out bitcoin all lower case; or use btc all lower case as well